### PR TITLE
Multilingual Constants Support Added -by Suvarna

### PR DESCRIPTION
**Pull request by suvarnakale**
Multilingual Constants Support Added -by Suvarna

### CODE REVIEW

File: src/app/admin/dlogin/dlogin.component.html [hunk 1]
```diff
@@ -4,56 +4,56 @@
...
 
 
         <div class="d-flex flex-row row mb-3">
             <div class="card p-4 col-sm-2 mr-2">
-                <p class="p16 mb-1 font-600">Facility Center</p>
-                <h2 class="font-600">0</h2>
-                <p class="p12 mb-1">RECORD</p>
-                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">View/Add</a>
+                <p class="p16 mb-1 font-600">{{'FACILITY_CENTER' | translate}}</p>
+                <h2 class="font-600">{{'0' | translate}}</h2>
```
Comment (Pratikshakhandagale) [inline src/app/admin/dlogin/dlogin.component.html:15]: @suvarnakale remove language constant for 0. its a value

**Comment by Pratikshakhandagale** [comment]
@suvarnakale Few comments added in your privious closed PR. please check it and resolved -  https://github.com/Sunbird-RC/Admin-Portal/pull/49#pullrequestreview-1200598458

File: src/assets/i18n/global/en-global.json [hunk 2]
```diff
@@ -249,8 +244,105 @@
...
+    "CREDENTIALING_FOR_ISSUERS_TO_CREATE_AND_ISSUE_CERTIFICATE" : "Credentialing for issuers to create and issue verified certificate",
+    "EXPERIANCE_DEMO" : "Experience Demo",
+    
+    "EDUCATION_RC" : "Education RC",
+    "WELCOME_TO_SUNBIRD_RC_BUILDER" : "Welcome to Sunbird RC builder",
+    "CREATE_YOUR_REGISTRY_CREDENIALLING_PLATFORM" : "Create your Registry and Credentialling Platform. ",
+    "NEED_HELP_WATCH_TRAINING_READ_DOCUMENTATION" : "Need help? Watch training or read documentation",
+    "ENTITY" : "Entity",
+    "FACILITY_CENTER" : "Facility Center",
+    "0" : "0",
```
Comment (Pratikshakhandagale) [inline src/assets/i18n/global/en-global.json:301]: @suvarnakale - remove 0 lang constant

File: src/app/admin/dlogin/dlogin.component.html [hunk 3]
```diff
@@ -4,56 +4,56 @@
...
+                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
             </div>
 
             <div class="card p-4 col-sm-2 mr-2">
-                <p class="p16 mb-1 font-600">Doctors</p>
-                <h2 class="font-600">0</h2>
-                <p class="p12 mb-1">RECORD</p>
-                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">View/Add</a>
+                <p class="p16 mb-1 font-600">{{'DOCTORS | translate'}}</p>
+                <h2 class="font-600">{{'0' | translate}}</h2>
```
Comment (Pratikshakhandagale) [inline src/app/admin/dlogin/dlogin.component.html:22]: @suvarnakale remove language constant for 0. its a value

File: src/app/admin/dlogin/dlogin.component.html [hunk 4]
```diff
@@ -4,56 +4,56 @@
...
+                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
             </div>
 
             <div class="card p-4 col-sm-2 mr-2">
-                <p class="p16 mb-1 font-600">Beneficiary</p>
-                <h2 class="font-600">0</h2>
-                <p class="p12 mb-1">RECORD</p>
-                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">View/Add</a>
+                <p class="p16 mb-1 font-600">{{'BENEFICIARY' | translate}}</p>
+                <h2 class="font-600">{{'0' | translate}}</h2>
```
Comment (Pratikshakhandagale) [inline src/app/admin/dlogin/dlogin.component.html:29]: @suvarnakale remove language constant for 0. its a value

File: src/app/admin/dlogin/dlogin.component.html [hunk 5]
```diff
@@ -4,56 +4,56 @@
...
+                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
             </div>
 
             <div class="card p-4 col-sm-2 mr-2">
-                <p class="p16 mb-1 font-600">Staff</p>
-                <h2 class="font-600">0</h2>
-                <p class="p12 mb-1">RECORD</p>
-                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">View/Add</a>
+                <p class="p16 mb-1 font-600">{{'STAFF' | translate}}</p>
+                <h2 class="font-600">{{'0' | translate}}</h2>
```
Comment (Pratikshakhandagale) [inline src/app/admin/dlogin/dlogin.component.html:36]: @suvarnakale remove language constant for 0. its a value

File: src/app/admin/dlogin/dlogin.component.html [hunk 6]
```diff
@@ -4,56 +4,56 @@
...
+                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">{{'VIEW_ADD' | translate}}</a>
             </div>
 
             <div class="card p-4 col-sm-2 mr-2">
-                <p class="p16 mb-1 font-600">Proof of Experience</p>
-                <h2 class="font-600">0</h2>
-                <p class="p12 mb-1">RECORD</p>
-                <a class="my-2 fs-14 btn btn-sec-bg submit-button align-self-start">View/Add</a>
+                <p class="p16 mb-1 font-600">{{'PROOF_OF_EXPERIANCE' | translate}}</p>
+                <h2 class="font-600">{{'0' | translate}}</h2>
```
Comment (Pratikshakhandagale) [inline src/app/admin/dlogin/dlogin.component.html:54]: @suvarnakale remove language constant for 0. its a value

File: src/assets/i18n/global/hi-global.json [hunk 7]
```diff
@@ -248,58 +244,101 @@
...
+  "PREVIEW": "पूर्वावलोकन",
+  "PRINT": "प्रिंट",
+  "VERIFY_CERTIFICATE" : "प्रमाणपत्र सत्यापित करें",
+"EDUCATION_RC" : "शिक्षा आरसी",
+  "WELCOME_TO_SUNBIRD_RC_BUILDER" : "सनबर्ड आरसी बिल्डर में आपका स्वागत है",
+  "CREATE_YOUR_REGISTRY_CREDENIALLING_PLATFORM" : "अपनी रजिस्ट्री और क्रेडेंशियल प्लेटफॉर्म बनाएं।",
+  "NEED_HELP_WATCH_TRAINING_READ_DOCUMENTATION" : "सहायता चाहिए? प्रशिक्षण देखें या दस्तावेज़ीकरण पढ़ें",
+  "ENTITY" : "एंटिटी",
+  "FACILITY_CENTER" : "सुविधा केंद्र",
+  "0" : "0",
```
[the remark this answers is not in the document]
Comment (suvarnakale) [reply src/assets/i18n/global/hi-global.json:314]: okay

**Pull request merged**